packages/@react-spectrum/color/intl/ar-AE.json

```diff
@@ -0,0 +1,10 @@
+{
+  "alpha": "Alpha",
```

Comment:
These files should not exist in the react-spectrum color package. They are in react-stately

Reply:
@dannify I've corrected that, and will fix in my loc script to avoid next time